Makefile

```diff
@@ -45,6 +46,7 @@ install:
 	install -Dm0644 target/$(TARGET)/$(FFI) "$(DESTDIR)$(libdir)/$(FFI)"
 	install -Dm0644 target/$(PKGCONFIG) "$(DESTDIR)$(libdir)/pkgconfig/$(PKGCONFIG)"
 	install -Dm0644 ffi/$(PACKAGE).h "$(DESTDIR)$(includedir)/$(PACKAGE).h"
+	install -Dm0644 "linux/$(TEST_LAUNCH_DESKTOP)" "$(DESTDIR)$(datadir)/applications/$(TEST_LAUNCH_DESKTOP)"
```

Comment:
Installing this unconditionally in the makefile may cause confusion for users of the AUR package or any other packaging outside of Pop.

You should be able to just use something like this in the `.install` file, without installing it here:

```
linux/com.system76.keyboardconfiguratorlaunchtest.desktop usr/share/applications
```

Reply:
[09246d4](https://github.com/pop-os/keyboard-configurator/pull/175/commits/09246d402b64b2584b2c1baeb5292b42f7cd7751)